plugin-modernizer-cli/src/test/java/io/jenkins/tools/pluginmodernizer/cli/MavenInvokerIntegrationTest.java

```diff
@@ -0,0 +1,22 @@
+package io.jenkins.tools.pluginmodernizer.cli;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import org.apache.maven.shared.invoker.*;
+import org.junit.jupiter.api.Test;
+import java.io.File;
+import java.util.Arrays;
+
+public class MavenInvokerIntegrationTest {
```

Comment:
By default to be run by failsafe IT must match a certain pattern: https://maven.apache.org/surefire/maven-failsafe-plugin/examples/inclusion-exclusion.html#inclusions